schema.sql

```diff
@@ -26,3 +26,24 @@ CREATE TABLE species (
 ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE ADD COLUMN species_id INTEGER REFERENCES species (id);
 ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
+-- Vet clinic database: add "join table" for visits
+
+CREATE TABLE vets(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50),
+    age INTEGER,
+    date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+    specialization_id SERIAL PRIMARY KEY,
+    vet_id INTEGER REFERENCES vets(id),
+    species_id INTEGER REFERENCES species(id)
+);
+
+CREATE TABLE visits (
```

Comment:
- [ ] It would be better if we add the `date` column to keep track of the `date of visits` as seen in the requirements
![Screenshot from 2023-09-29 00-17-48](https://github.com/Nessrine88/Vet-clinic-database/assets/46388113/2167220f-e36e-4c8d-839f-25d9eb91f880)